modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jspf

```diff
@@ -129,23 +129,27 @@ if (cpAttachmentFileEntry != null) {
 		});
 	});
 
-	function <portlet:namespace />saveAttachmentFileEntry(forceDisable) {
-		var form = window.document['<portlet:namespace />fm'];
+	Liferay.provide(
+		window,
+		'<portlet:namespace />saveAttachmentFileEntry',
+		function (forceDisable) {
```

Comment:
This `forceDisable` is an arg that doesn't seem like it's used anywhere. Can we remove it? @garofalog